lib/googleauth/credentials_loader.rb

```diff
@@ -134,6 +134,7 @@ def from_system_default_path(scope = nil)
       def warn_if_cloud_sdk_credentials(client_id)
         warn CLOUD_SDK_CREDENTIALS_WARNING if client_id == CLOUD_SDK_CLIENT_ID
       end
+      module_method :warn_if_cloud_sdk_credentials
```

Comment:
Should this be `module_function`?

Reply:
Huh. You're totally right. It should. Why the heck didn't this crash when I tested it? Anyway, fixed.